src/Help.tsx

```diff
@@ -59,6 +59,14 @@ const Help: FC = () => {
             { color, start: 'b4', end: 'f4' },
           ]}
         />
+
+        <HelpBoard
+          title='5. Threaten mate in one'
```

Comment:
Let's change this to "Threats of mate in one" to match the noun phrasing of the other items.